<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="rick_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>526</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/rick_0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>681</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="MainGrid" rowminimumheight="0" columnminimumwidth="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="move_left">
       <property name="text">
        <string>&lt;----</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shoot">
       <property name="text">
        <string>Shoot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="move_right">
       <property name="text">
        <string>----&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="salir_boton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="usuario_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>move_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>move_left_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>move_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>move_right_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shoot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>shoot_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>salir_boton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>salir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>695</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>shoot_signal()</signal>
  <signal>move_character_signal(str)</signal>
  <slot>move_left_clicked()</slot>
  <slot>move_right_clicked()</slot>
  <slot>shoot_clicked()</slot>
  <slot>salir()</slot>
 </slots>
</ui>
